tests/TestCase/ORM/QueryRegressionTest.php

```diff
@@ -1605,7 +1605,7 @@ public function testEagerLoadOrderAndSubquery()
     public function testFormatResultsMemoryLeak()
     {
         $this->loadFixtures('Articles', 'Authors', 'Tags', 'ArticlesTags');
-        $this->skipIf(env('CODECOVERAGE') === 1, 'Running coverage this causes this tests to fail sometimes.');
+        $this->skipIf(env('CODECOVERAGE'), 'Running coverage this causes this tests to fail sometimes.');
```

Comment:
Do we need to (bool) cast this env() result?

Reply:
No need to get too pedantic in tests :slightly_smiling_face:. Just checking whether the var is defined is enough for deciding whether to skip the test.

Reply:
I am just thinking about skipIf() getter a bool typehint.. :) Then this breaks with any other casted boolish value.

Reply:
Done